metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          'Apache-2.0'
 description      'Installs/Configures redis'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '2.6.1'
+version          '2.7.0'
```

Comment:
I am new to this project and I unfortunately do not see any contributing file but generally it is best to avoid changing the version in the PR unless the community specifically asks for it for several reasons:
- PRs are not a FIFO queue, if your contribution gets merged after PR you would have to rebase and fix
- maintainer disagrees with your interpretation on the impact of the change therefore the version (in this case it is correct)
- some projects do not version every contribution and make regularly scheduled releases lumping in multiple changes

Reply:
I can remove the version bump, but:

- I dont mind rebasing
- I think every change adds value therefore every change constitutes a release. Releases should be incremental and include as little change as possible. Then if bugs are reported by users it will be easy for a user to determine which version works and which does not and then solving the bug will be easier because a smaller version diff will include the breaking change.

Just my two cents.

Reply:
rebased and now the version is up to what i was bumping to. so there are no metadata.rb changes now as requested.

Reply:
I personally agree with you that every change should be released and versioned and this is how I run the `sensu-plugins` projects and am slowly working on pushing that out across the other sensu projects but the reality is that this adds extra overhead to the project and right now from what I can tell the current maintainers are not really giving this the love it needs. Also I don't mind asking users to rebase or rebasing myself but a lot of users do not know how to properly rebase meaning either I need to create a new PR and cherry pick or checkout their forks branch and rebase on their behalf this adds extra overhead to maintenance.

I will personally be versioning and releasing any *functional* prs within 24 hours of acceptance for any PR I review but I can't force other maintainers to take this view point.